tests/test_Step.cc

```diff
@@ -324,7 +324,8 @@ TEST_CASE("execute(): Lua exceptions", "[Step]")
         }
         catch(const Error& e)
         {
-            REQUIRE_THAT(e.what(), StartsWith("Script execution error: pippo"));
+            REQUIRE_THAT(e.what(), Contains("Script execution error"));
+            REQUIRE_THAT(e.what(), Contains("pippo"));
```

Comment:
This is, unfortunately, not an improvement. The stack trace contains the failing line of the script, so with the `Contains` it is not guaranteed that "pippo" actually made it out as the error message:
```
stack traceback:
        [C]: in function 'base.error'
        [string "function boom(); error('pippo', 0); end; boom..."]:1: in function
  'base.boom'
        [string "function boom(); error('pippo', 0); end; boom..."]:1: in main
```

Reply:
Fixed while keeping the `Contains`. I believe it is better to have a more loose string based test that does not need concrete lineups.

(This message vanished when I pushed 'resolve'.. )